### PR TITLE
Set default keras backend at runtime

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: tensorflow.spec [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ### RPM external tensorflow 2.16.1
+### INITENV SETV KERAS_BACKEND tensorflow
```
Comment (smuzaffar) [inline tensorflow.spec]: @iarspider , previously this was add in the keras.file ( https://github.com/cms-sw/cmsdist/pull/9388/commits/1ba91fcf806f9fd4068e54ff36832bc0d0624482 ) . I do not remember which packages actually needed this ... do you? Was it one of `tfaot` package? Though it does not make any sense but I think it is safe to move this to `pip/keras.file`

Comment (iarspider) [reply tensorflow.spec]: Sure.

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test for CMSSW_14_2_TF_X/el8_amd64_gcc12

**Comment by iarspider** [comment]
please test for CMSSW_14_2_TF_X/el8_aarch64_gcc12

**Comment by smuzaffar** [comment]
looks good to go in TF IB

**Pull request merged**